packages/preferences/src/browser/folder-preference-provider.ts

```diff
@@ -28,12 +28,13 @@ export interface FolderPreferenceProviderFactory {
 export const FolderPreferenceProviderOptions = Symbol('FolderPreferenceProviderOptions');
 export interface FolderPreferenceProviderOptions {
     folder: FileStat;
+    fileName: string;
```

Comment:
could we call it kind, i.e.
```ts
   kind: 'settings' | 'launch'
```

file name is an implementation detail of a folder preference provider of a concrete kind

Reply:
fixed